share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+Should only be used when performance is improved for a frequently called bit of code.
+.It Sy __null_sentinel Ta
+.It Sy __exported Ta
+.It Sy __hidden Ta
+.It Sy __printflike(fmtarg,firstvararg) Ta
+.It Sy __scanflike(fmtarg,firstvararg) Ta
+.It Sy __format_arg(f) Ta
+.It Sy __strfmonlike(fmtarg,firstvararg) Ta
+.It Sy __strtimelike(fmtarg,firstvararg) Ta
+.It Sy __printf0like(fmtarg,firstvararg) Ta
+.It Sy __strong_reference(sym,aliassym) Ta
+.It Sy __weak_reference(sym,alias) Ta
+.It Sy __warn_references(sym,msg) Ta
+.It Sy __sym_compat(sym,impl,verid) Ta
+.It Sy __sym_default(sym,impl,verid) Ta
+.It Sy __GLOBAL(sym) Ta
+.It Sy __WEAK(sym) Ta
+.It Sy __DECONST(type, var) Ta
+.It Sy __DEVOLATILE(type, var) Ta
+.It Sy __DEQUALIFY(type, var) Ta
+.It Sy __RENAME(x) Ta
+.El 
+.Ss Emulated Keywords
+As C evolves, many of the old macros we once used have been incorporated into
+the standard language.
+As this happens, we add support for these keywords as macros for older
+compilation environments.
+Sometimes this results in a nop in the older environment.
+.Bl -column ".Sy Keyword" ".Sy Description"
+.It Sy Keyword Ta Sy Description
+.It Sy _Alignas(x) Ta
+.It Sy _Alignof(x) Ta
+.It Sy _Noreturn Ta Expandsd to
+.Dq [[noreturn]]
+in C++-11 and newer compilation environments, otherwise
+.Dq __dead2
+.It _Static_assert(x, y) Ta Compile time assertion that
+.Fa x
+is true, otherwise emit
+.Fa y
+as the error message.
+.It Sy _Thread_local Ta Designate variable as thread local storage.
+.It Sy __generic Ta implement _Generic-like features which aren't entirely possible to emulate the _Generic keyword.
+.It Sy __noexcept Ta to emulate the C++11 argument-less noexcept form.
+.It Sy __noexcept_if Ta to emulate the C++11 conditional noexcept form.
+.El
+.Ss Support Macros
+The following macros are defined, or have specific values, to denote certain
+things about the build environment.
+.Bl -column ".Sy Macro" ".Sy Description"
+.It Sy Macro Ta Sy Description
+.It Sy __LONG_LONG_SUPPORTED Ta Variables may be declared
+.Dq long long .
+This is defined for C99 or newer and C++ environments.
+.It Sy __STDC_LIMIT_MACROS Ta
+.It Sy __STDC_CONSTANT_MACROS Ta
+.El
+.Ss Nice to have macros
+.Bl -column ".Sy Macro" ".Sy Description"
+.It Sy Macro Ta Sy Description
+.It Sy __offsetof(type,field) Ta
+.It Sy __rangeof(type,start,end) Ta
+.It Sy __containerof(x,s,m) Ta
+.El
+.Ss ID Strings
+This section is deprecated, but is kept around because too much contrib software
+still uses these.
+.Bl -column ".Sy Macro" ".Sy Description"
+.It Sy Macro Ta Sy Description
+.It Sy __IDSTRING(name,string) Ta
+.It Sy __FBSDID(s) Ta
+.It Sy __RCSID(s) Ta
+.It Sy __RCSID_SOURCE(s) Ta
+.It Sy __SCCSID(s) Ta
+.It Sy __COPYRIGHT(s) Ta
+.El
+.Sh Supported C Environments
+FreeBSD supports a number C standard environments.
+Selection of the language dialect is a compiler dependent command line option,
+though it is usually
+.Fl cstd=XX
+where XX is the standard you desire, like c89 or c23.
```

Comment:
```suggestion
where XX is a standard such as c89 or c23.
```

Reply:
```suggestion
where XX is a standard like c89 or c23.
```

Reply:
why a -> the. It's the one specific standard you'll be using... I'm wanting specifically to emphasize that while we support many standards, you use only one at a time.